app/src/main/res/layout/ods_text_button.xml

```diff
@@ -60,7 +60,7 @@
             app:text="Button"
             app:enabled="true"
             app:odsTextButtonStyle="primary"
-            app:displaySurface="standard" />
+            app:inverseTheme="false" />
```

Comment:
This line can be removed like you did for other buttons.

Same comment for `ods_buttoon.xml`.